apps/io-func-wallet-solution-backend/src/infra/trust-anchor/index.ts

```diff
@@ -42,7 +43,9 @@ export class EidasTrustAnchor implements TrustAnchor {
       new URL(oidFederation, this.#configuration.trustAnchorUri.href),
       (metadataUrl) => metadataUrl.href,
       getRequest(this.fetchWithTimeout),
-      TE.map(jose.decodeJwt),
+      TE.chain((value) =>
+        TE.tryCatch(async () => jose.decodeJwt(value), E.toError)
```

Comment:
Are you sure you need `async`?

Reply:
Fixed in https://github.com/pagopa/io-wallet/pull/23/commits/f4fc20b9071428897999b36aa1a4d4f1e0730226